qiita_db/metadata_template/test/test_util.py

```diff
@@ -265,6 +265,16 @@ def test_load_template_to_dataframe_with_nulls(self):
         exp.index.name = 'sample_name'
         assert_frame_equal(obs, exp, check_like=True)
 
+    def test_load_template_to_dataframe_better_tokenizing_error_msg(self):
+        with self.assertRaises(RuntimeError) as error:
+            qdb.metadata_template.util.load_template_to_dataframe(
+                StringIO('sample_name\tcollection_timestamp\n'
+                         '2.Sample1\t2014-05-29 12:24:51\t\n'
+                         '2.Sample2\taaa\n'
+                         'xxx\tadfa\t\t\n'))
+        self.assertIn('Your file has more columns with values than headers.',
+                      str(error.exception))
```

Comment:
Any reason not to use `assertRaisesRegex`? 

```suggestion
        with self.assertRaisesRegex(RuntimeError, 'Your file has more columns with values than headers'):
            qdb.metadata_template.util.load_template_to_dataframe(
                StringIO('sample_name\tcollection_timestamp\n'
                         '2.Sample1\t2014-05-29 12:24:51\t\n'
                         '2.Sample2\taaa\n'
                         'xxx\tadfa\t\t\n'))
```